unicorn.js

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2017 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+module.exports = {
+  plugins: [
+    'unicorn',
+  ],
+  rules: {
+    // Improve regexes by making them shorter, consistent, and safer
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/better-regex.md
+    'unicorn/better-regex': 'error',
+
+    // ****** Enforce a specific parameter name in catch clauses
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/catch-error-name.md
+    'unicorn/catch-error-name': 'error',
+
+    // Enforce passing a message value when throwing a built-in error
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/error-message.md
+    'unicorn/error-message': 'error',
+
+    // Require escape sequences to use uppercase values
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/escape-case.md
+    'unicorn/escape-case': 'error',
+
+    // Add expiration conditions to TODO comments
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/expiring-todo-comments.md
+    'unicorn/expiring-todo-comments': 'error',
+
+    // Enforce explicitly comparing the length property of a value
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/explicit-length-check.md
+    'unicorn/explicit-length-check': 'error',
+
+    // Enforce importing index files with .
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/import-index.md
+    'unicorn/import-index': 'error',
```

Comment:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/import-index.md#ignoreimports

Node 14 supports `{ "type": "module" }` in the `package.json` and it has a few differences compared to ES syntax code being bundled... do you think we need this? It would lead to cleaner paths when importing, maybe a warning?